packages/autocomplete/src/examples/multiselect.md

```diff
@@ -0,0 +1,304 @@
+The `MultiSelect` layout operates similarly to the `Autocomplete`, but must
+manage the interaction between the Input and it's collection of Tags.
+
+This examples includes an example of "copy-to-clipboard" functionality for tags.
+
+```jsx
+const { MenuView, Item, AddItem, Separator } = require('@zendeskgarden/react-menus');
+const { FauxInput, Input, TextGroup, Label } = require('@zendeskgarden/react-textfields');
+const { Tag, Close } = require('@zendeskgarden/react-tags');
+const { Anchor } = require('@zendeskgarden/react-buttons');
+const { KEY_CODES, FieldContainer } = require('@zendeskgarden/react-selection');
+
+const NoItemsMessage = styled.div`
+  margin: 16px;
+  text-align: center;
+`;
+
+const SpacedTag = styled(Tag)`
+  margin: 2px;
+`;
+
+const InlineInput = styled(Input)`
+  flex-grow: 1;
+  margin-left: 2px;
```

Comment:
This doesn't layout correctly for RTL. You might need `margin: 0 2px;`

Reply:
Good catch. Now fixed.